src/tests/stdlib/join.output.txt

```diff
@@ -0,0 +1 @@
+apple,banana,cherry
```

Comment:
Shouldn't it be like this:

```suggestion
apple, banana, cherry
```

Since we join with comma followed by space `, `